Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -440,6 +440,7 @@ extension PostCampaignCheckoutViewController: STPApplePayContextDelegate {
     let transactionId = payment.token.transactionIdentifier
 
     let params = ApplePayParams(
+      paymentMethodId: paymentMethod.stripeId,
```

Comment:
I think it would make more sense to pass the `paymentMethodId` as its own thing to the view model, especially if we don't need it for normal pledges. It's not used where the other params are used, and in fact, if I remember correctly, we don't actually use the `ApplePayParams` in the late pledge flow anymore anyways (they're just not cleaned up properly since this was part of that quick patch for fixing apple pay). It'd be nice but not strictly necessary to clean up the apple pay params as part of this work & only pass the info the view model needs.

Reply:
You are correct, we no longer need any info from the `ApplePayParams` in the VM: https://github.com/kickstarter/ios-oss/blob/main/Library/ViewModels/PostCampaignCheckoutViewModel.swift#L348